src/components/Capture/ProfileData.tsx

```diff
@@ -122,6 +113,16 @@ const ProfileData = ({
           />
         ))}
       </ValidationProvider>
+      <Button
+        onClick={handleSubmit}
+        className={classNames(
+          theme['button-centered'],
+          theme['button-lg'],
+          style['submit-button']
+        )}
+      >
+        {translate('profile_data.button_submit')}
+      </Button>
```

Comment:
Just wondering if it is sitting across the bottom of the page rightly if we take it out of screen layout

Reply:
It is no longer, no.

E.g.:
<img width="536" alt="Screenshot 2022-04-28 at 11 44 20" src="https://user-images.githubusercontent.com/20243687/165735813-5ec8425d-4e94-4dc6-a7aa-d97ab262e3f7.png">

<img width="533" alt="Screenshot 2022-04-28 at 11 43 55" src="https://user-images.githubusercontent.com/20243687/165735762-7e0c6478-a39f-4043-92d2-99d0fdfe099f.png">

<img width="543" alt="Screenshot 2022-04-28 at 11 44 30" src="https://user-images.githubusercontent.com/20243687/165735850-c7830c8e-171c-4d2a-a301-26ee2e6bcfc5.png">

Reply:
as long as it is the same in the design then.